js/components/controlBar.js

```diff
@@ -772,28 +601,21 @@ var ControlBar = React.createClass({
 
         return (
           <div key={CONSTANTS.CONTROL_BAR_KEYS.AUDIO_AND_CC} className="oo-popover-button-container">
-            <AccessibleButton
-              ref={function(e) {
+            <ControlButton
+              {...commonButtonProps}
+              onRef={function(e) {
                 this.setToggleButtons(CONSTANTS.MENU_OPTIONS.MULTI_AUDIO, e);
               }.bind(this)}
               style={selectedStyle}
               className={multiAudioClass}
-              focusId={CONSTANTS.FOCUS_IDS.MULTI_AUDIO}
+              focusId={CONSTANTS.CONTROL_BAR_KEYS.AUDIO_AND_CC}
```

Comment:
Was it deliberate to change the focusId name to match the icon name? Just curious

Reply:
Yes, the focus id just needs to be any unique id, so we never really needed to have a separate set of ids for this purpose. I recently added the `CONTROL_BAR_KEYS` values, that's why we weren't using these before. It's simpler if there's a single id per button.